data_generator/theBigDataGen_cql.js

```diff
@@ -55,7 +55,7 @@ const generateFavoriteListings = () => {
 
 // generate data for home_listings_by_id table
 const writeHomeListingData = fs.createWriteStream('./data_generator/csv/cql_home_listings_data.csv');
-writeHomeListingData.write('listing_id,price,size_bd,size_ba,size_sqft,street_address,neighborhood,listing_image,favorite, similar_homes\n');
+writeHomeListingData.write('listing_id|price|size_bd|size_ba|size_sqft|street_address|neighborhood|listing_image|favorite|similar_homes\n');
```

Comment:
is there a specific reason you chose pipe delimiters as opposed to commas?

Reply:
yes, the neighborhood has commas in it, for example: "Cambridgeshire, Murrayport, OH". Also, similar_homes is a map data type which consists of comma delimited key:value pairs.